### PR TITLE
Auto generate CLI options in docs

### PR DESCRIPTION
**Pull request by chao2zhang**
There may be some text discrepancies between the existing `cli.md` and the annotation in `CliArgs.kt`.

### CODE REVIEW

File: docs/_data/sidebars/home_sidebar.yml [hunk 1]
```diff
@@ -38,6 +38,9 @@ entries:
     - title: with the Command Line Interface
       url: /cli.html
       output: web
+    - title: Command Line Interface Options
+      url: /cli-options.html
+      output: web
```
Comment (BraisGabin) [inline docs/_data/sidebars/home_sidebar.yml:43]: Should this be in getting started? I think that it fits better in user documentation.

**Review by BraisGabin — APPROVED** [review]
I like this! Is there a way in jekyll to include one markdown inside another? It could be great to keep this in the same document I think.

Anyway, we can merge this because the important thing here is to have a reliable documentation.

File: docs/_data/sidebars/home_sidebar.yml (hunk 1 above)
Comment (chao2zhang) [reply docs/_data/sidebars/home_sidebar.yml:43]: I have successfully used "include" in jekyll. So I will revert this change.

**Comment by chao2zhang** [comment]
> I like this! Is there a way in jekyll to include one markdown inside another? It could be great to keep this in the same document I think.
> 
> Anyway, we can merge this because the important thing here is to have a reliable documentation.

The only limitation is that we cannot include files in the parent path `../` according to https://jekyllrb.com/docs/includes/#including-files-relative-to-another-file

**Review by BraisGabin — APPROVED** [review]
👏🏻👏🏻👏🏻

**Pull request merged**